src/Tribe/Timezones.php

```diff
@@ -29,6 +29,17 @@ public static function init() {
 	protected static function invalidate_caches() {
 		add_filter( 'pre_update_option_gmt_offset', array( __CLASS__, 'clear_site_timezone_abbr' ) );
 		add_filter( 'pre_update_option_timezone_string', array( __CLASS__, 'clear_site_timezone_abbr' ) );
+		add_action( 'update_option_gmt_offset', [ __CLASS__, 'clear_timezone_cache' ] );
+		add_action( 'update_option_timezone_string', [ __CLASS__, 'clear_timezone_cache' ] );
```

Comment:
:no_entry_sign: Short array syntax is not allowed

<sub>hash: 3ce9a92baba6ce23e4e2dea2783acb5e</sub>